apix-impl/src/test/java/eu/xenit/apix/tests/metadata/RecursiveParentsBaseUnitTest.java

```diff
@@ -0,0 +1,47 @@
+package eu.xenit.apix.tests.metadata;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import eu.xenit.apix.alfresco.ApixToAlfrescoConversion;
+import org.alfresco.service.ServiceRegistry;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.repository.NodeService;
+import org.alfresco.service.cmr.security.AccessStatus;
+import org.alfresco.service.cmr.security.PermissionService;
+import org.junit.Before;
+
+public abstract class RecursiveParentsBaseUnitTest {
```

Comment:
Thanks for making the effort for making this run as unit test and not integration test :+1: